benchmarks/src/utils.rs

```diff
@@ -234,8 +234,12 @@ where
         .expect("Verification failed");
     if let Some(leaf_vm_config) = leaf_vm_config {
         let leaf_vm_pk = leaf_keygen(app_config.leaf_fri_params.fri_params, leaf_vm_config);
-        let leaf_prover = LeafProver::new(leaf_vm_pk, app_pk.leaf_committed_exe);
-        leaf_prover.generate_proof(&app_proof);
+        let leaf_prover = VmLocalProver::<SC, NativeConfig, BabyBearPoseidon2Engine>::new(
+            leaf_vm_pk,
+            app_pk.leaf_committed_exe,
+        );
+        let leaf_controller = LeafProvingController { num_children: 1 };
```

Comment:
a little weird this is being hardcoded here: previously it was from some constructor, or at least using the DEFAULT_* constant

Reply:
oh this is in benchmark: I think there might be a parameter in `args`